Dockerfile

```diff
@@ -0,0 +1,27 @@
+FROM ubuntu
+
+# Install wget, unzip, python2.7
+RUN apt-get update \
+  && apt-get install -y wget \
+  && apt-get install -y unzip \
+  && apt-get install -y python2.7 python-pil -y \
+  && apt-get install -y python-pip \
+  && rm -rf /var/lib/apt/lists/*
+
+# Install GAE
+RUN wget -O /appengine.zip https://storage.googleapis.com/appengine-sdks/featured/google_appengine_1.9.38.zip
+RUN unzip /appengine.zip -d /appengine && rm /appengine.zip
+
+
+# Copy src files to /app
+RUN mkdir /app
+COPY . /app
```

Comment:
So I mainly made the file with test/production in mind. I like the idea of not having to continuously rebuild the container, but I don't like the idea of polluting the docker host with the src outside of this container. I wish there was a way to make the Dockerfile smart enough to say in dev use volumes and in test/prod copy the src.